bugwarrior/docs/services/youtrack.rst

```diff
@@ -0,0 +1,103 @@
+YouTrack
+========
+
+You can import tasks from your YouTrack instance using
+the ``youtrack`` service name.
+
+Example Service
+---------------
+
+Here's an example of a YouTrack target::
+
+    [my_issue_tracker]
+    service = youtrack
+    youtrack.host = youtrack.example.com
+    youtrack.login = turing
+    youtrack.password = 3n1Gm@
+
+The above example is the minimum required to import issues from
+YouTrack.  You can also feel free to use any of the
+configuration options described in :ref:`common_configuration_options`
+or described in `Service Features`_ below.
+
+Service Features
+----------------
+
+Host Options
++++++++++++++++++++++
+
+The ``youtrack.host`` field is used to construct a URL for
+the YouTrack server. It defaults to a secure connection scheme (HTTPS)
+on the standard port (443).
+
+Connecting on a different port::
+
+    youtrack.port = 8443
+
+Allow server to use a self-signed SSL certificate::
+
+    youtrack.verify_ssl = False
```

Comment:
In terms of consistency within the Bugwarrior project, the option as implemented matches Jira and Gitlab:
  https://github.com/ralphbean/bugwarrior/search?utf8=%E2%9C%93&q=verify_ssl&type=Code

* From the perspective of whether or not the option should exist, vs requiring a trust-store update, I'm deferring to those services precedent.
* Implementation of a custom certfile parameter may be best served as a separate feature request that could then be applied uniformly to all three major issue trackers.

In terms of the documentation, that can be reworded if desired. The [Jira](https://github.com/ralphbean/bugwarrior/blob/develop/bugwarrior/docs/services/jira.rst) docs for `verify_ssl` are perhaps more 'dangerous' in its suggestion to disable it in response to any verification error. The [Gitlab](https://github.com/ralphbean/bugwarrior/blob/develop/bugwarrior/docs/services/gitlab.rst) docs use a more clinical description. I aimed for something in-between with YouTrack.

In the end it comes down to Bugwarrior's target audience ... particularly users of YouTrack/Gitlab/Jira which are traditonally self-hosted solutions. Persons using a local install are unlikely to be using SSL, and in a corporate environment it's unlikely you'd encounter a self-signed certificate unless the workstations were IT managed and had said certificates imported. So in my opinion users that need to use this option would know what they are doing, either because of their technical background, or because they setup the target issue tracker with the self-signed certificate to begin with.

I'm happy to elaborate on the security risks for this option in the docs if you think it's appropriate to the project's scope though.